language/move-lang/functional-tests/tests/libra/natives/event_cannot_emit_resource.move

```diff
@@ -0,0 +1,13 @@
+script {
+use 0x1::Event;
+use 0x1::LBR::LBR;
+use 0x1::Libra;
+
+fun main(account: &signer) {
+    let handle = Event::new_event_handle<Libra::Libra<LBR>>(account);
+    let zero = Libra::zero<LBR>();
+    Event::emit_event<Libra::Libra<LBR>>(&mut handle, zero);
+    Event::destroy_handle<Libra::Libra<LBR>>(handle);
+}
+}
+// check: MoveSourceCompilerError resource type not copyable
```

Comment:
If we leave in this test, it needs to be an exp file. Or it needs to be moved out of functional tests

Reply:
Similarly, this covered in `constraints_not_satisfied_type_arguments_internal_call.move`